src/win32/fs.cpp

```diff
@@ -1259,6 +1259,25 @@ fsfp_t WinDirNotify::fsfingerprint()
 #endif
 }
 
+bool WinDirNotify::fsstableids() const
```

Comment:
@mattw-mega Would you mind testing this on windows? Set up a sync to a FAT drive and ensure this function returns false? Thanks!